src/pages/namespace/Settings/Broadcasts/index.tsx

```diff
@@ -0,0 +1,227 @@
+import {
+  Table,
+  TableCell,
+  TableHead,
+  TableHeaderCell,
+  TableRow,
+} from "~/design/Table";
+
+import { BroadcastsSchemaType } from "~/api/broadcasts/schema";
+import { Card } from "~/design/Card";
+import { Checkbox } from "~/design/Checkbox";
+import { FC } from "react";
+import { Radio } from "lucide-react";
+import { TableBody } from "@mui/material";
+import { useBroadcasts } from "~/api/broadcasts/query/useBroadcasts";
+import { useTranslation } from "react-i18next";
+import { useUpdateBroadcasts } from "~/api/broadcasts/mutate/updateBroadcasts";
+
+const leftColClasses = "col-span-2";
+const labelCellClasses = "place-self-center px-2";
+const switchCellClasses = "place-self-center px-2";
+const rowClasses = "grid grid-cols-5 gap-0 lg:pr-8 xl:pr-12";
+
+const Broadcasts: FC = () => {
+  const { t } = useTranslation();
+  const { data } = useBroadcasts();
+  const { mutate } = useUpdateBroadcasts();
+
+  if (!data?.broadcast) return null;
+
+  const broadcasts = data.broadcast;
+
+  const toggleBroadcast = (key: keyof BroadcastsSchemaType) =>
+    mutate({
+      payload: {
+        broadcast: {
+          [key]: !broadcasts[key],
+        },
+      },
+    });
+
+  return (
+    <>
+      <div className="mb-3 flex flex-row justify-between">
+        <h3 className="flex items-center gap-x-2 font-bold text-gray-10 dark:text-gray-dark-10">
+          <Radio className="h-5" />
+          {t("pages.settings.broadcasts.title")}
+        </h3>
+      </div>
+
+      <Card>
+        <Table>
+          <TableHead>
+            <TableRow className={rowClasses}>
```

Comment:
I was wondering if we disable the hover effect on the table header rows? I don't know if it's just me, but I only expect this hover effect to be available on rows that I can interact with.


If you agree, it should work with this code (I just used it myself)

```typescript
import { twMergeClsx } from "~/util/helpers"; // this will work when you update the base branch of your PR
...
<TableRow
  className={twMerge(
    rowClasses,
    "hover:bg-inherit dark:hover:bg-inherit"
  )}
>


```

Reply:
Agreed and fixed :-)